server/connector/connector.go

```diff
@@ -194,22 +272,45 @@ func (c *connector) Post(w http.ResponseWriter, req *http.Request) {
 		} else {
 			http.Error(w, fmt.Sprintf(`{"error":"unknown error: %s"}`, err.Error()), http.StatusInternalServerError)
 		}
+
 		return
 	}
 	go c.Run(subscriber)
 	c.logger.WithField("topic", topic).Info("Subscription created")
 	fmt.Fprintf(w, `{"subscribed":"/%v"}`, topic)
+
+	if errFill == nil {
+		err = event.report(c.KafkaProducer, c.KafkaReportingTopic)
```

Comment:
I've added a message at startup and also made a new check.If the misconfiguration is found no logs will be printed.